src/main/java/gregtech/api/util/GTUtility.java

```diff
@@ -452,6 +452,21 @@ public static byte getTierByVoltage(long voltage) {
         return tierByVoltage.ceilingEntry(voltage).getValue();
     }
 
+    /**
+     * <strong>Do not</strong> use for calculation. Intended for displays in multiblock machines and similar.
+     * <p>
+     * Ex: This method turns 960EU/t and 480EU/t into HV.
+     *
+     * @return lowest tier that can handle passed voltage for display
+     */
+    public static byte getTierForVoltageDisplay(long voltage) {
+        byte tierDown = getTierByVoltage(voltage / 2);
+        byte tier = getTierByVoltage(voltage);
+        //noinspection RedundantIfStatement
+        if (tierDown != tier) return tierDown;
+        return tier;
```

Comment:
Honestly I'd rather see this done the same way as you did for `getTierByVoltage` using its own map. That way you can mimic the exact same behavior as it was previously.